CHANGELOG.md

```diff
@@ -28,6 +28,17 @@
         * `Iterable<Byte>.decodeAsn1VarUInt()`
         * `ByteArray.decodeAsn1VarUInt()`
 * Revamp implicit tagging
+* Revamp `Asn1Element.parse()`
+  * `Asn1Element.parse()` semantics remain the same
```

Comment:
is this needed in a _change_-log?

Reply:
rephrased slightly, so it makes sense to have everything listed